### PR TITLE
feat(smoke): simple smoke tests after deploy

### PR DESCRIPTION
**Pull request by blacha**

### CODE REVIEW

File: packages/smoke/src/health.test.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  it('should 200 when GET /v1/version', async () => {
+    const res = await ctx.req('/v1/version');
+    assert.equal(res.status, 200);
+    const data = await res.json();
+
+    assert.equal(typeof data.version, 'string');
+    assert.ok(data.version);
+
+    assert.equal(typeof data.hash, 'string');
+    assert.ok(data.hash);
```
Comment (blacha) [inline packages/smoke/src/health.test.ts:25]: Future Work: Validate the hash is the same as the current git hash.

File: .github/workflows/push.yml [hunk 2]
```diff
@@ -60,6 +60,12 @@ jobs:
           GOOGLE_ANALYTICS: ${{secrets.GOOGLE_ANALYTICS_NON_PROD}}
           NODE_ENV: "dev"
 
+      - name: (NonProd) Deploy
```
Comment (blacha) [inline .github/workflows/push.yml:63]: Name needs to change
> (NonProd) Deploy - Smoke Test

File: .github/workflows/push.yml [hunk 3]
```diff
@@ -60,6 +60,12 @@ jobs:
...
+        if: github.ref == 'refs/heads/master' && github.repository == 'linz/basemaps'
+        run: |
+            cd packages/smoke
+            node --test
```
Comment (blacha) [inline .github/workflows/push.yml:67]: Future Work: Use Lambda URL directly not hitting cloud front? or use cache busting??

Prefer cache busting, so we are testing the entire stack.

File: packages/smoke/src/health.test.ts [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import assert from 'node:assert';
+import { describe, it } from 'node:test';
+import { ctx } from './base.js';
+
+describe('health', () => {
+  it('should 200 when GET /v1/ping', async () => {
+    const res = await ctx.req('/v1/ping');
+    assert.equal(res.status, 200);
```
Comment (blacha) [inline packages/smoke/src/health.test.ts:8]: validate cache miss too.

`x-cache == null || x-cache == Miss from cloudfront`

**Review by Wentao-Kuang — APPROVED** [review]

**Review by Wentao-Kuang — APPROVED** [review]

**Pull request merged**